### ISSUE TITLE
apitrace fails to work due to using non-unicode variants of certain winapi functions

### ISSUE THREAD

**Issue by galtgendo** [apitrace/apitrace#728]
I'm writing this from memory as I've used apitrace only a couple times in the last few years.

This was only seen in wine, I have no idea if this fails the same way with native Windows.

Anyway, if apitrace is placed in a path that's outside locale constrains of the locale it's run in, it fails to run due to using GetCurrentDirectoryA.

I don't quite recall how exactly the failure gets triggered. The general setup was roughly that (as I use apitrace that rarely) I've been placing apitrace in a subdirectory of the app being tested. Now, the failure was either when apitrace was run from its directory and app path was relative or the other way around - one of those worked, the other didn't. The failure originated from a codepage conversion error on the path.

**Comment by jrfonseca** [comment]
Yeah, It's a hassle but apitrace should indeed be using unicode interfaces..
